# srv/GlobalBoundaryCenter.srv
---
geometry_msgs/Point center
bool valid  